crypto/types/types.go

```diff
@@ -22,9 +22,3 @@ type PrivKey interface {
 type (
 	Address = tmcrypto.Address
 )
-
-// IntoTmPubKey allows our own PubKey types be converted into Tendermint's
-// pubkey types.
-type IntoTmPubKey interface {
```

Comment:
sounds good. A centralized function `PubKeyToProto` to make the conversion SDK-TM pubkeys is probably a cleaner way to do it.